packages/gator-permissions-snap/scripts/generate-manifest.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable n/no-process-env */
```

Comment:
Could we also make it so that we have a single generate-manifest script in shared package? An would via parameters determine directory so that we don't have duplicated code?